common/hooks/pre-install/00-libdir.sh

```diff
@@ -1,7 +1,9 @@
 # This hook creates the wordsize specific libdir symlink.
 
 hook() {
-	if [ "${pkgname}" != "base-files" ]; then
+	if [ -L ${PKGDESTDIR}/usr/lib${XBPS_TARGET_WORDSIZE} ]; then
+		return 0
+	elif [ "${pkgname}" != "base-files" ]; then
```

Comment:
This should only happen if you're rebuilding in a dirty destdir... Maybe it should have a dotfile that tracks the hook already having been run?

Reply:
Yes, but my use case is:
```sh
$ ./xbps-src pkg foo
$ do-some-test-and-hate-the-result
$ fix-some-stuff
$ ./xbps-src install foo
$ inspect $DESTDIR
$ ./xbps-src -f pkg foo # this steps will run into error :-[p

Reply:
This is a hack, not a proper fix, I think.

Reply:
Maybe leave a comment about it being a hack? The issue with these hooks is that some of them we'd like to run only once, and others we'd like to run despite it having been run already, so a general `.pre-pkg-hook-*` mechanism isn't enough :/

Still, I thought `ln -sf` avoid issues with creating symlinks inside symlinks.

Reply:
Yay, the keyword is _thought_